flake8_trio/visitors/visitor103_104.py

```diff
@@ -16,19 +16,27 @@
 from .helpers import critical_except, error_class, iter_guaranteed_once
 
 _trio103_common_msg = "{} block with a code path that doesn't re-raise the error."
+_suggestion = " Consider adding an `except {}: raise` before this exception handler."
+_suggestion_dict = {
+    "trio": "trio.Cancelled",
+    "anyio": "anyio.get_cancelled_exc_class()",
+}
+_suggestion_dict[
+    "[trio|anyio]"
+] = f'[{_suggestion_dict["trio"]}|{_suggestion_dict["anyio"]}]'
+_suggestion_dict["[anyio|trio]"] = _suggestion_dict["[trio|anyio]"]
```

Comment:
Let's canonicalize this, so we only have to worry about one ordering.

I'd also recommend a tuple literal rather than the (not-valid-at-runtime) `|`.

Reply:
I fiddled a bit with this and ... maybe the code is better now? Idk, I'll likely fiddle around with this more when I write more library-specific/dependent code